python/tk_framework_desktopserver/shotgun/scripts/cache_commands.py

```diff
@@ -218,6 +227,7 @@ def cache(cache_file, data, base_configuration, engine_name, config_data):
         arg_data["base_configuration"],
         arg_data["engine_name"],
         arg_data["config_data"],
+        arg_data["config_is_mutable"],
```

Comment:
I would rather abbreviate both instances of the word configuration or not abbreviate both here.
`base_configuration`/`configuration_is_mutable` or `base_config` and `config_is_mutable`